CoreFunctionality/src/au/gov/asd/tac/constellation/functionality/whatsnew/whatsnew.txt

```diff
@@ -1,6 +1,9 @@
 == 3030-12-31 Getting Started
 <p>If you're new to CONSTELLATION, read the <a href="" helpId="au.gov.asd.tac.constellation.functionality.introduction">introduction</a>.</p>
 
+== 2020-04-20 Option to remove a saved preference
```

Comment:
A minor comment @mimosa2 but can the title be changed to "Data Access View"?

Reply:
Ok. Thanks. I pushed the change just now.